src/lib/viewers/controls/annotations/annotations-button-region-tooltips/PersistentOnboardingBoxEditAnnotationsTooltip.tsx

```diff
@@ -0,0 +1,81 @@
+import React, { useContext, PropsWithChildren } from 'react';
+import classNames from 'classnames';
+
+// TODO: Compare bundle sizes and possibly re-write here
+import { TargetedClickThroughGuideTooltip } from 'box-ui-elements/es/features/targeting';
+
+import { TargetingApi } from '../../../../types';
+
+import { ControlsLayerContext } from '../../controls-layer';
+import { DependenciesContext } from '../../dependencies';
+
+import './PersistentOnboardingBoxEditAnnotationsTooltip.scss';
+
+export type Props = PropsWithChildren<{
+    isEnabled?: boolean;
+}>;
+
+export default function PersistentOnboardingBoxEditAnnotationsTooltip({
+    children,
+    isEnabled = false,
+}: Props): JSX.Element {
+    const { experiences, isLocaleLanguageEnglish, onPersistentOnboardingTooltipBack } = useContext(DependenciesContext);
+    const { setIsForced } = useContext(ControlsLayerContext);
+
+    const { persistentOnboardingBoxEditAnnotations: persistentOnboardingBoxEditAnnotationsExperience } = experiences;
+    const useTargetingApi = (): TargetingApi => ({
+        ...persistentOnboardingBoxEditAnnotationsExperience,
+        onClose: (): void => {
+            persistentOnboardingBoxEditAnnotationsExperience.onClose();
+            setIsForced(false);
+        },
+        onComplete: (): void => {
+            persistentOnboardingBoxEditAnnotationsExperience.onComplete();
+            setIsForced(false);
+        },
+        onShow: (): void => {
+            persistentOnboardingBoxEditAnnotationsExperience.onShow();
+            setIsForced(true);
+        },
+    });
+
+    const title = __('persistent_onboarding_box_edit_annotations_tooltip_title');
+    const body = __('persistent_onboarding_box_edit_annotations_tooltip_body');
+
+    const image = (
+        <div
+            className={classNames('bp-PersistentOnboardingBoxEditAnnotationsTooltip-image', {
+                isLocalized: isLocaleLanguageEnglish,
+            })}
+            data-testid="tooltipImage"
+        />
+    );
+
+    const primaryButtonProps = {
+        children: __('persistent_onboarding_box_edit_annotations_tooltip_button_next'),
+    };
+
+    const backButtonProps = {
+        children: __('persistent_onboarding_box_edit_annotations_tooltip_button_previous'),
+        onClick(): void {
+            onPersistentOnboardingTooltipBack();
```

Comment:
The targeting API should give us many different callbacks. Can we use those rather than passing in a net new one?

Reply:
Does the Targeting API need to evolve to support this new capability? It seems like it could be a common need.

Reply:
I'm not 100% sure about that, but I think we currently don't have any plans of integrating such a feature into our Targeting API. @wpiesiak @megansmith-box Can you confirm?

It's not a bad idea though. It definitely feels limiting our Experiences API allows you to proceed forward with the experience but not to go back.

Reply:
I think we'd like each experience to be as self-contained as possible. Let's solve this in the framework rather than adding yet more options to the Preview SDK.

Reply:
> I think we'd like each experience to be as self-contained as possible. 

I agree. That's why my first approach was to not put this experience-related code into `box-content-preview` at all. But that turned out to be impossible.

> Let's solve this in the framework rather than adding yet more options to the Preview SDK.

I don't see a way to do this within the current framework. Do you?

Reply:
My ask is to see if we can extend the framework with new functionality for this new behavior.

Reply:
I extended `TargetingApi` type with `onPrevious()` callback, as we discussed on Slack. Am resolving this. Please create a new thread to discuss the new implementation.